app/main.js

```diff
@@ -915,6 +923,47 @@ function showNotification (text) {
   })
 }
 
+function runCommand (event, argv, workingDirectory) {
+  const args = argv.slice(app.isPackaged ? 1 : 2)
+  const cmd = new Command(args, app.getVersion())
+
+  // if this command is already executed by the second-instance, return early
+  if (!cmd.checkInMain()) return
+
+  switch (cmd.command) {
+    case 'reset':
```

Comment:
Can we `log.info` for each option? So in case I am debugging something I know  where events come from

Reply:
https://github.com/papey/stretchly/commit/f7df5cb82d9df19aed4e12d846aa48d2fae0285c